java-bootcamp-bahia-t1/src/MusicPlayer/MusicPlayerState.java

```diff
@@ -0,0 +1,7 @@
+package MusicPlayer;
+
+public interface MusicPlayerState {

```

Comment:
Please could you write two lines explaining which pattern did you use to modelate item 6? and Why did you choose it?